railties/lib/rails/commands/server/server_command.rb

```diff
@@ -114,7 +114,7 @@ class ServerCommand < Base # :nodoc:
         desc: "Runs server as a Daemon."
       class_option :using, aliases: "-u", type: :string,
         desc: "Specifies the Rack server used to run the application (thin/puma/webrick).", banner: :name
-      class_option :pid, aliases: "-P", type: :string, default: DEFAULT_PID_PATH,
```

Comment:
This change affected to an output of help. Please add a default value to desc like `port` and `binding`.